.github/workflows/ci.yml

```
@@ -91,7 +91,11 @@ jobs:
 
   build_cargo:
     name: Build (cargo)
-    runs-on: ubuntu-latest
+    build_matrix:
+      strategy:
+        matrix:
+          os: [ubuntu-latest, macos-latest, windows-latest]
```

```suggestion
    strategy:
      matrix:
        os: [ubuntu-latest, macos-latest, windows-latest]
```

https://github.com/DeFiCh/metachain/actions/runs/3546915406